src/components/landing-card/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+
+import CardGeostory from '@/components/landing-card/geostories';
+import CardMonitor from '@/components/landing-card/monitors';
+
+const Card: FC<{
+  id: string;
+  title: string;
+}> = ({ title, id, ...data }) => {
+  const isMonitor = id.startsWith('m');
+  const isGeostory = id.startsWith('g');
```

Comment:
This was not your best idea, if you need a way to identify them let's request to the API.